fineract-provider/src/main/java/org/apache/fineract/cob/api/ConfigureBusinessStepResource.java

```diff
@@ -69,6 +71,20 @@ public class ConfigureBusinessStepResource {
     private final ConfigJobParameterService configJobParameterService;
     private final PortfolioCommandSourceWritePlatformService commandWritePlatformService;
 
+    @GET
+    @Path("/names")
+    @Produces({ MediaType.APPLICATION_JSON })
+    @Operation(summary = "List Business Jobs", description = "Returns the configured Business Jobs")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(array = @ArraySchema(schema = @Schema(implementation = ConfigureBusinessStepResourceSwagger.GetBusinessJobConfigResponse.class)))) })
+    public String retrieveAllConfiguredBusinessJobs(@Context final UriInfo uriInfo) {
+
+        List<String> businessJobNames = configJobParameterService.getAllJobNames();
+        final Gson gson = new Gson();
```

Comment:
We're not using GSON directly to serialize API data. Please look at existing other API implementations.

Reply:
This is just a String array, with the configured job names, Do I need to add the serializer?